File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/qute-example/java/src/test/java/org/acme/qute/HelloResourceTest.java
package org.acme.qute;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.containsString;

import org.junit.jupiter.api.Test;

import io.quarkus.test.junit.QuarkusTest;

@QuarkusTest
public class HelloResourceTest {

    @Test
    public void testEndpoint() {
        given()
                .when().get("/qute/hello")
                .then()
                .statusCode(200)
                .body(containsString("<p>Hello world!</p>"));

        given()
                .when().get("/qute/hello?name=Lucie")
                .then()
                .statusCode(200)
                .body(containsString("<p>Hello Lucie!</p>"));
    }

}

File: devtools/platform-descriptor-json/src/main/resources/codestarts/qute-example/kotlin/src/test/kotlin/org/acme/qute/HelloResourceTest.kt
package org.acme.qute

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.containsString
import org.junit.jupiter.api.Test

@QuarkusTest
class HelloResourceTest {
    @Test
    fun testEndpoint() {
        given()
                .`when`().get("/qute/hello")
                .then()
                .statusCode(200)
                .body(containsString("<p>Hello world!</p>"))
        given()
                .`when`().get("/qute/hello?name=Lucie")
                .then()
                .statusCode(200)
                .body(containsString("<p>Hello Lucie!</p>"))
    }
}